src/jdk.compiler/share/classes/com/sun/tools/javac/file/BaseFileManager.java

```diff
@@ -529,6 +525,6 @@ synchronized void newOutputToPath(Path path) throws IOException {
 
         // Check whether we've already opened this file for output
         if (!outputFilesWritten.add(realPath))
-            log.warning(LintCategory.OUTPUT_FILE_CLASH, Warnings.OutputFileClash(path));
+            log.warning(LintWarnings.OutputFileClash(path)); // @@@: shouldn't we check for suppression?
```

Comment:
Re: "shouldn't we check for suppression?": If `-Xlint:-output-file-clash` is in effect, then `outputFilesWritten` will be null and you'll never get here.